projects/plugins/jetpack/_inc/client/recommendations/prompts/site-type/index.jsx

```diff
@@ -104,7 +106,10 @@ const SiteTypeQuestionComponent = props => {
 				'jetpack'
 			) }
 			answer={ answerSection }
-			illustrationPath="recommendations/site-type-illustration.jpg"
+			illustrationPath={
+				! canShowProductSuggestions ? 'recommendations/site-type-illustration.jpg' : null
+			}
+			sidebarCard={ canShowProductSuggestions ? <DiscountCard /> : null }
```

Comment:
Should the `discount` and `expiryDate` props be getting passed into `DiscountCard`?

Reply:
As mentioned in my other comment, `DiscountCard` shouldn't be part of this PR. A rebase seemed to have fixed it.